build-farm/set-platform-specific-configurations.sh

```diff
@@ -24,8 +24,8 @@ then
     export CONFIGURE_ARGS_FOR_ANY_PLATFORM="${CONFIGURE_ARGS_FOR_ANY_PLATFORM} --disable-warnings-as-errors"
 fi
 
-# jdk-19 and above support reproducible builds
-if [[ "${JAVA_FEATURE_VERSION}" -ge 19 ]]
+# jdk-17 and above support reproducible builds
+if [[ "${JAVA_FEATURE_VERSION}" -ge 19 ] || [ "${JAVA_FEATURE_VERSION}" -eq 17 ]]
```

Comment:
Surely this would work?

```suggestion
if [[ "${JAVA_FEATURE_VERSION}" -ge 17 ]]
```

Reply:
comment was incorrect